EventFilter/L1TRawToDigi/plugins/implementations_stage2/EMTFBlockRPC.cc

```diff
@@ -55,6 +55,15 @@ namespace l1t {
         uint16_t RPCc = payload[2];
         uint16_t RPCd = payload[3];
 
+        // Run 3 has a different EMTF DAQ output format since August 26th
```

Comment:
What is the purpose of this commented out block? Please either add a comment which explains its usage, or remove it

Reply:
This was my bad. I forgot to remove that part :). Should be fixed now